http/src/routing/route_display.rs

```diff
@@ -2414,6 +2420,16 @@ mod tests {
         assert_eq!(route.display().to_string(), "users/@me");
     }
 
+    #[test]
+    fn test_get_current_user_guild_member() {
+        let route = Route::GetCurrentUserGuildMember { guild_id: GUILD_ID };
+
+        assert_eq!(
+            route.display().to_string(),
+            format!("users/@me/guilds/{guild_id}/member", guild_id = GUILD_ID)
+        )
```

Comment:
Let's get this loking like the other tests
```suggestion
        let route = Route::GetCurrentUserGuildMember { guild_id: GUILD_ID };
        assert_eq!(
            route.display().to_string(),
            format!("users/@me/guilds/{guild_id}/member", guild_id = GUILD_ID)
        )
```